Tasks/ServiceFabricComposeDeploy/ServiceFabricComposeDeploy.ps1

```diff
@@ -82,7 +82,7 @@ try {
 
     if ($registryCredentials -ne "None")
     {
-        if (-not $isEncrypted)
+        if (-not $isEncrypted -and $clusterConnectionParameters["ServerCertThumbprint"])
```

Comment:
This is not enough to prevent existing builds from failing as Get-ServiceFabricEncryptedText will fail if the passwrod can't be encrypted, this used to be a warning. Best thing to do is bump the major version.